app/frontend/types.ts

```diff
@@ -59,8 +60,21 @@ export type DerivationScheme = {
   keyfileVersion: string
 }
 
-export type Transaction = {}
+type Coins = {getCoin: string}
+type Input = [_Address, Coins]
+type Output = [_Address, Coins]
+
 export type HexString = string
+export type Transaction = {
+  ctbId: HexString
+  ctbTimeIssued: number
+  ctbInputs: Array<Input>
+  ctbOutputs: Array<Output>
+  ctbInputSum: Coins
+  ctbOutputSum: Coins
+  fee: Lovelace
+  effect: Lovelace
+}
```

Comment:
These are all defined in `explorer-types.ts`.

Reply:
I missed that, thanks.